superset/viz.py

```diff
@@ -2015,6 +2015,9 @@ def get_data(self, df):
             ],
         }
 
+        southWest = [df[fd.get('all_columns_x')].min(), df[fd.get('all_columns_y')].min()]
```

Comment:
NIT: snake over camel case in python

Reply:
NIT (optional): you could save some hash lookups by going `x_series = df[fd.get('all_columns_x')]` first